### DISCUSSION TITLE
Blogging over SSH!

### DISCUSSION THREAD

**Discussion by deemkeen** [charmbracelet/inspo#27] [General]
https://github.com/deemkeen/stegodon

stegodon is a simple, single user blog tui-application written in Golang, using the wonderful [Charm Tools](https://github.com/charmbracelet). It allows users to create notes, as well as view them in a clean and easy to navigate console interface. It is also possible to subscribe to these notes via RSS and to federate via ActivityPub (WIP!) Contributions are very welcome!

**Comment by allyring** [comment]
Neat! Really awesome project, and definitely at the point where you could submit a PR to get it added to the repo.

**Reply by deemkeen** [reply]
Thanks a lot, looking forward for collaboration! :)

**Comment by ktprograms** [comment]
This is quite interesting. Is there any way to view other users' posts, or is the only way through the master feed?

**Reply by deemkeen** [reply]
Thank you, indeed it is now possible, to get notes from a certain user you have to query for the username like this: `http://127.0.0.1:9999/feed?username=<username>`

**Comment by bashbunni** [comment]
Awesome idea!!
